source/extensions/transport_sockets/common/passthrough.cc

```diff
@@ -4,11 +4,17 @@
 #include "envoy/network/transport_socket.h"
 
 #include "source/common/buffer/buffer_impl.h"
+#include "source/common/network/transport_socket_options_impl.h"
 
 namespace Envoy {
 namespace Extensions {
 namespace TransportSockets {
 
+void PassthroughFactory::hashKey(std::vector<uint8_t>& key,
+                                 Network::TransportSocketOptionsConstSharedPtr options) const {
+  return Network::TransportSocketOptionsUtility::commonHashKey(key, options);
```

Comment:
Should this implementation go into a base class, since it is used in almost every transport socket?

Reply:
Good idea. Thanks for the feedback.

Reply:
Implemented a common class.